### DISCUSSION TITLE
esp32 wifi-bluetooth?

### DISCUSSION THREAD

**Discussion by chipset2011** [zepp-health/zeppos-forum#51] [Q&A]
Hello, let's say I have an esp32 wifi module that sends small data over wifi to a client-server, is there any way to get this data from the server over the wifi network ? Or is there a way to interact with the same esp32 via bluetooth directly?

**Comment by MyoungXUE** [comment]
Hello, theoretically it is possible to accomplish this idea of yours.

First of all, you need to understand the architecture of Mini Program https://docs.zepp.com/docs/guides/architecture/arc/, with the function of accessing the network, the specific implementation can refer to our example https://docs.zepp.com/docs/samples/app/fetchAPI/.
